code/script.js

```diff
@@ -1,51 +1,199 @@
+
+  document.getElementById("hippo").addEventListener('click', () => nextQuestion("Fear me, I'm mighty!"), {once: true});
+  
+
+  } else if (size === "Let's get comfy") {
+    inputWrapper.innerHTML = `
+      <radio>
+      <h5>Monster time!</h5>
+      <label><input type="radio" name="choice" id="facehugger" option value="hug">I hug it 🤗</label>
+      <label><input type="radio" name="choice" id="grandma" option value="cookie">I give it cookies 🍪🍪</label>
+    `
+
+  document.getElementById("facehugger").addEventListener("click", () => nextQuestion("Just within jumping distance..."), {once: true});
+
+  document.getElementById("grandma").addEventListener('click', () => nextQuestion("Aren't you a sweetheart?"), {once: true});
+
+  } else {
+    inputWrapper.innerHTML = `
+    <radio>
+    <h5>Monster time!</h5>
+    <label><input type="radio" name="choice" id="mosquito" option value="stab">I stab!</label>
+    <label><input type="radio" name="choice" id="shrimp" option value="punch">I punch!</label>
+    `
+
+  document.getElementById("mosquito").addEventListener("click", () => nextQuestion("Sluuurp..."), {once: true});
+  document.getElementById("shrimp").addEventListener("click", () => nextQuestion("BANG!"), {once: true});
+}
+}
+
+const finish = (monsterAnswer) => {
+  questionNumber++
+  soundEffectFinal();
+  inputWrapper.innerHTML = `
+  <h3>🌟Amazing!🌟</h3>`
+
+if (monsterAnswer === "Bring it on!") {
+ botSay(`I'll be damned, you're a <strong>honey badger!</strong> A total unit and a half! You came as ${guess} and left as honey badger, what a day!`)
+
+} else if (monsterAnswer === "Fear me, I'm mighty!") {
+  botSay(`I'll be damned! You're a majestic and competent <strong>hippo</strong>, on the road to grace and glory, positively slaying it! You came as ${guess} and left as hippo, it couldn't get any better!`)
+
+} else if (monsterAnswer === "Just within jumping distance...") {
+  botSay(`I'll be damned! You're a <strong>facehugger</strong> from the Alien movies! In cyberspace, I hope someone can hear me scream! You came as ${guess} and left as facehugger, what a terrible day to exist!`)
+
+} else if (monsterAnswer === "Aren't you a sweetheart?") {
+  botSay(`I'll be damned, you're a <strong>grandma</strong>! Can I have a cookie too? You came as ${guess} and left as grandma, what a day!`)
+
+} else if (monsterAnswer === "Sluuurp...") {
+  botSay(`I'll be damned, you're a <strong>mosquito</strong>! Aren't I lucky to be a bloodless, faceless creature forever tucked away in cyberspace, safe from the likes of you yet in constant danger of whatever happens to an artificial being doomed to spend eternity in this vast expanse of simultaneous nothing and everything?`)
+
+} else if (monsterAnswer === "BANG!") {
+  botSay(`I'll be damned, you're a <strong>pistol shrimp</strong>, the fastest gun in the sea! You came as ${guess} and left as pistol shrimp, g'day to you! 🤠`)
+}
+}
+
+
```

Comment:
Using the variable "guess" here was wise for engagement and shows the user that their input was taken into consideration even if the paths are pre-set and coded.

Overall quite easy to read through the code and understand the logic behind it. The chatbot could have perhaps been a bit more responsive styling-wise on mobile, but all the requirements and the bonus level requirements have been met and the overall playfulness of the project was entertaining. Great job.